readme.md

```diff
@@ -14,8 +14,8 @@ Note: `solidity-docgen` [does not work on Windows](https://github.com/OpenZeppel
 ```sh
 git clone https://github.com/aragon/hack.git
 cd hack/website
-npm install
-npm start
+yarn install
```

Comment:
Since we're doing this, let's also update [this last `npm` remnant in the workflow](https://github.com/aragon/hack/blob/master/.github/workflows/ci.yml#L30)